### ISSUE TITLE
Randomness of sumo environments

### ISSUE THREAD

**Issue by JINKEHE** [INFLUENCEorg/aienvs#35]
If I didn't miss anything, currently, to control the randomness of simulation, sumo gym adaptor uses a fixed random seed 42 to build up simulations, including the generation of vehicles for every step. Unless the user explicitly calls env.seed(x) where x is another random seed, whatever environments he/she builds or resets will be using the same random seed.

I think this is highly undesirable. I suggest the following behaviour:

Seed will be given as part of the parameters to build the environment, whose default value is None.
If it is None, essentially there is no randomness control of the environment, which will give us randomized training and testing environments. This would be the default behaviour. 
We can still do randomness control to make experiments reproducible by giving the random seeds explicitly.

**Comment by czechows** [comment]
I overlooked that, sorry. You mean simply changing the default from reproducible to random? That makes sense, check with @miguelsuau who is using Sumo for his projects and go ahead with the modification.

**Comment by miguelsuau** [comment]
Yes, I agree. This is something I also saw when I was working with the new version in aienvs. If no seed is provided then the default should be random.

**Comment by JINKEHE** [comment]
do you think that's the thing that caused your trained agent to perform worse in this new version of aienvs?
